no-vary-query.md

```diff
@@ -0,0 +1,264 @@
+# The `No-Vary-Query` HTTP response header
+
+Caching is useful for making web pages load faster, and thus creating better user experiences. Prominent caches on the web platform include the HTTP cache, as well as the subjects of this repository, the prefetch and prerender caches.
+
+One of the most important cache keys for web resources is the resource's URL. However, sometimes multiple URLs can represent the same resource. This leads to caches not always being as helpful as they could be: if the browser has the resource cached under one URL, but the resource is then requested under another, the cached version will be ignored.
+
+This proposal tackles a specific subset of this general problem, for when multiple URLs for the same resource differ according to their URL's query components. Via a new HTTP header, `No-Vary-Query`, resources can declare to the browser that some or all parts of the query can be ignored for cache matching purposes. For example, if the order of the query parameters should not cause cache misses, this is indicated using
```

Comment:
nit: "their URL's" technically expands to "multiple URLs' URL's", and I'd suggest emphasizing that the URLs differ only in query, i.e, "...for when a resource has multiple URLs which differ only in certain query components".

nit: elide "to the browser", since any HTTP cache implementation could also process this in theory; while transparent caching proxies are uncommon these days, caching CDNs are common and offer similar functionality.